src/MeetingsSDKAdapter.js

```diff
@@ -635,6 +635,14 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       const sdkMeeting = this.fetchMeeting(ID);
 
       if (sdkMeeting.passwordStatus === 'REQUIRED') {
+        if (!(options.password || options.hostKey)) {
```

Comment:
if we dont pass password or hostkey and the meeting requires password then we need not call verify and return from the function saying  passwordRequired: true so that widget shows popup/modal

Reply:
Thanks for the explanation.